From ff73e5d0169b5490a7c45a62d69a6fb57de5da6e Mon Sep 17 00:00:00 2001
From: Jason Wu <jason.wu@petalogix.com>
Date: Thu, 1 Nov 2012 20:15:55 +1000
Subject: [PATCH 446/628] net: ethernet: axi_emac: Fix of_irq_to_resource
 compilation error

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit 9b332e4fde5a4c5a5a56814582f8e6bee0704c74

Include of_irq.h header to fix implicit declaration of function
'of_irq_to_resource' compilation error when target machine is Zynq

Signed-off-by: Jason Wu <huanyu@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/net/ethernet/xilinx/xilinx_axienet.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/net/ethernet/xilinx/xilinx_axienet.h b/drivers/net/ethernet/xilinx/xilinx_axienet.h
index 44b8d2b..ee73048 100644
--- a/drivers/net/ethernet/xilinx/xilinx_axienet.h
+++ b/drivers/net/ethernet/xilinx/xilinx_axienet.h
@@ -9,6 +9,7 @@
 #define XILINX_AXIENET_H
 
 #include <linux/netdevice.h>
+#include <linux/of_irq.h>
 #include <linux/spinlock.h>
 #include <linux/interrupt.h>
 
-- 
1.7.5.4

